app/modules/enduser_setup.c

```diff
@@ -1525,4 +1538,4 @@ static const LUA_REG_TYPE enduser_setup_map[] = {
   { LNILKEY, LNILVAL}
 };
 
-NODEMCU_MODULE(ENDUSER_SETUP, "enduser_setup", enduser_setup_map, NULL);
+NODEMCU_MODULE(ENDUSER_SETUP, "enduser_setup", enduser_setup_map, NULL);
```

Comment:
I'm not sure whats happening on this line, but I doubt that the change is needed or intended.